scripts/globals/job_utils/ranger.lua

```diff
@@ -273,6 +273,12 @@ xi.job_utils.ranger.useBountyShot = function(player, target, ability, action)
     local playerTHLevel     = player:getMod(xi.mod.TREASURE_HUNTER)
     local newTHLevel        = 0
 
+    -- base animation was for gun, -1 = archery
+    -- Note: hume male's archery animation is bugged and looks like shadowbind
```

Comment:
We could probably also put this comment in the file for bounty shot - just to hammer it home. Lol SE